packages-ts/starknet/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './utils'
+export * as account from './funder'
```

Comment:
We shouldn't alias like this. I suggest just rename the file to `account.ts`

```suggestion
export * as account from './account'
```